experiments/browser_based_querying/patches/graphql-language-service@5.1.7.patch

```diff
@@ -1,21 +1,51 @@
 diff --git a/esm/interface/getAutocompleteSuggestions.js b/esm/interface/getAutocompleteSuggestions.js
-index ac159d132d095a789b966627d315f4238c79ab60..a3be2d27c9d5c7d6fd3745093301a07d28e79a8d 100644
+index ac159d132d095a789b966627d315f4238c79ab60..a5c69725101bdf5919e27cd55c42e231813b0543 100644
 --- a/esm/interface/getAutocompleteSuggestions.js
 +++ b/esm/interface/getAutocompleteSuggestions.js
-@@ -234,7 +234,12 @@ export function getAutocompleteSuggestions(schema, queryText, cursor, contextTok
+@@ -63,6 +63,9 @@ const hasTypeSystemDefinitions = (sdl) => {
+     }
+     return hasTypeSystemDef;
+ };
++const primitives = new Set(['Int', 'Float', 'String', 'Boolean'])
++const propertyDirectives = new Set(['fold', 'recurse', 'optional', 'transform'])
++const edgeDirectives = new Set(['filter', 'tag', 'output', 'transform'])
```

Comment:
These don't seem like they are in the right categories. Are the names `propertyDirectives` and `edgeDirectives` backwards?

Reply:
[bb3fb60](https://github.com/obi1kenobi/trustfall/pull/490/commits/bb3fb604678ab10f2d039088d716e28c0677bd41)